src/style/form/style-form.css

```diff
@@ -0,0 +1,24 @@
+.form-container {
+  min-height: 72px;
+  max-width: 360px;
+  margin-left: 10px;
+  padding: 12px;
```

Comment:
Nos outros comentários que eu fiz sobre esse estilo deve ficar mais claro tbm, mas lembra que quem for usar o campo de formulário pode cuidar do espaçamento e ta largura que esses campos terão. Então isso não precisa ser tratado aqui dentro do componente. Faz sentido?

Reply:
sim, porém o meu form ficou com uma largura enorme eu coloquei um (width: 50%;) e voltou a largura normal

Reply:
Mas a ideia é justamente que ele pegue toda a largura disponível pra ele, e quem cuide desse tamanho seja uma div pai (que será colocada nas páginas). Todo o formulário que vc for construir numa página deve ter os mesmo tamanhos, então é melhor vc criar uma div pai com 50% de width (ou até com 360px mesmo) do que colocar essa width em todos os campos, concorda?

Reply:
concordo